client/src/main/scala/drt/client/components/FaqsPage.scala

```diff
@@ -0,0 +1,227 @@
+package drt.client.components
+
+import diode.data.Pot
+import drt.client.modules.GoogleEventTracker
+import drt.client.services.SPACircuit
+import drt.shared.AirportConfig
+import japgolly.scalajs.react.component.Scala.Component
+import japgolly.scalajs.react.vdom.html_<^._
+import japgolly.scalajs.react.{Callback, CtorType, ScalaComponent}
+
+import scala.scalajs.js
+
+sealed trait FaqsPage {
+
+  def style(allFaqsShow: Boolean) = if (allFaqsShow) js.Dictionary("display" -> "block") else js.Dictionary("display" -> "none")
+}
+
+object AllFaqsPage extends FaqsPage {
+
+  case class Props(showSection: String)
+
+  val component: Component[Props, Unit, Unit, CtorType.Props] = ScalaComponent.builder[Props]("FAQs")
+    .render_P(props =>
+      props.showSection match {
+        case "DeskAndQueues" => <.div(^.className := "faqs-class", <.h3("FAQs"), DeskAndQueuesFaqsComponent(true), ArrivalsFaqsComponent(), PortConfigurationFaqsComponent(), StaffMovementsFaqsComponent(), MonthlyStaffingFaqsComponent())
+        case "arrivals" => <.div(^.className := "faqs-class", <.h3("FAQs"), DeskAndQueuesFaqsComponent(), ArrivalsFaqsComponent(true), PortConfigurationFaqsComponent(), StaffMovementsFaqsComponent(), MonthlyStaffingFaqsComponent())
+        case "portConfiguration" => <.div(^.className := "faqs-class", <.h3("FAQs"), DeskAndQueuesFaqsComponent(), ArrivalsFaqsComponent(), PortConfigurationFaqsComponent(true), StaffMovementsFaqsComponent(), MonthlyStaffingFaqsComponent())
+        case "staff-movements" => <.div(^.className := "faqs-class", <.h3("FAQs"), DeskAndQueuesFaqsComponent(), ArrivalsFaqsComponent(), PortConfigurationFaqsComponent(), StaffMovementsFaqsComponent(true), MonthlyStaffingFaqsComponent())
+        case "monthly-staffing" => <.div(^.className := "faqs-class", <.h3("FAQs"), DeskAndQueuesFaqsComponent(), ArrivalsFaqsComponent(), PortConfigurationFaqsComponent(), StaffMovementsFaqsComponent(), MonthlyStaffingFaqsComponent(true))
+        case _ => <.div(^.className := "faqs-class", <.h3("FAQs"), DeskAndQueuesFaqsComponent(), ArrivalsFaqsComponent(), PortConfigurationFaqsComponent(), StaffMovementsFaqsComponent(), MonthlyStaffingFaqsComponent())
+      }
+      ,
+    )
+    .componentDidMount(_ => Callback {
+      GoogleEventTracker.sendPageView(s"faqs")
+    })
+    .build
+
+  def apply(showSection: String): VdomElement = component(Props(showSection))
+}
+
+case class FaqsModel(airportConfig: Pot[AirportConfig])
+
+object DeskAndQueuesFaqsComponent extends FaqsPage {
```

Comment:
looks like you could quite easily make this into a reusable component which takes a list of FAQs, what do you think?

Reply:
May be , Let me see .